include/pipettes/firmware/interfaces.hpp

```diff
@@ -63,22 +63,22 @@ auto get_interrupt_queues<PipetteType::THREE_EIGHTY_FOUR_CHANNEL>()
 
 namespace linear_motor {
 
-auto get_interrupt(pipette_motor_hardware::MotorHardware& hw,
+auto get_interrupt(motor_hardware::MotorHardware& hw,
                    LowThroughputInterruptQueues& queues)
     -> MotorInterruptHandlerType<linear_motor_tasks::QueueClient>;
-auto get_interrupt(pipette_motor_hardware::MotorHardware& hw,
+auto get_interrupt(motor_hardware::MotorHardware& hw,
                    HighThroughputInterruptQueues& queues)
     -> MotorInterruptHandlerType<linear_motor_tasks::QueueClient>;
-auto get_motor_hardware(motor_configs::LowThroughputPipetteMotorHardware pins)
-    -> pipette_motor_hardware::MotorHardware;
-auto get_motor_hardware(motor_configs::HighThroughputPipetteMotorHardware pins)
-    -> pipette_motor_hardware::MotorHardware;
-auto get_motion_control(pipette_motor_hardware::MotorHardware& hw,
+auto get_motor_hardware(motor_hardware::HardwareConfig pins)
```

Comment:
I'm changing a couple of these getter functions in this file to use `motor_hardware` class objects instead of `pipette_motor_hadware`- this shouldn't negatively affect anything else for the pipettes, right?